library/pdf_creator.class.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * PdfCreator class
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2017 Jerry Padgett <sjpadgett@gmail.com>
+ * @license https://www.gnu.org/licenses/agpl-3.0.en.html GNU Affero General Public License 3
+ */
+use Knp\Snappy\Pdf;
+use Symfony\Component\Debug\ExceptionHandler;
+
+class PdfCreator
+{
+
+    private $binaryPath;
+
+    private $tempPath;
+
+    private function getBinaryPath()
+    {
+        global $webserver_root;
+        // This will not necessarily reflect actual machine bus width but php bus size.
+        $intsize = strlen(decbin(~ 0));
+        if (empty(strstr(php_uname("m"), '64'))) {
+            $bit = "32bit";
+        } else {
+            $bit = "64bit";
+        }
+        try {
```

Comment:
@sjpadgett, Note that since your a member of OpenEMR group, you can create repos, so feel free to do this as needed for this work. Very exciting; openemr is possibly about to have its own custom composer module.